crates/client/src/client/schema/node_info.rs

```diff
@@ -25,6 +25,7 @@ pub struct NodeInfo {
     pub max_tx: U64,
```

Comment:
It would be nice to also add max gas and max size in bytes of the tx pool

Reply:
I added it in the general structure like `max_tx` currently exists. I feel like it should be on a sub structure either like `PoolStats` or a new one `PoolConfiguration` but as we want to backport this to the existing version I don't like breaking the existing field `max_tx` by moving it to a sub structure.